### ISSUE TITLE
num_workers: 1的情况下ch_ppocr_v1.1训练  [python3] <defunct>

### ISSUE THREAD

**Issue by yisampi** [PaddlePaddle/PaddleOCR#1751]
num_workers: 1的情况下ch_ppocr_v1.1检测和识别训练都会  [python3] <defunct>

**Comment by WenmuZhou** [comment]
麻烦描述详细一点，如训练命令，环境这些，尽可能的提供可复现的脚本

**Comment by yisampi** [comment]
@WenmuZhou 
环境
paddlepaddle-gpu   1.8.5.post107 
Python 3.7.7
一块32G的GPU 
训练命令
python tools/train.py -c configs/det/det_r18_vd_db_bankcard_v5_no_check_label.yml -o Global.pretrain_weights=./pretrain_models/ch_ppocr_server_v1.1_det_train/best_accuracy 

$ cat configs/det/det_r18_vd_db_bankcard_v5_no_check_label.yml
Global:
  algorithm: DB
  use_gpu: true
  epoch_num: 1200
  log_smooth_window: 20
  print_batch_step: 2
  save_model_dir: ./output/det_r_18_vd_db_bankcard_v5_no_check_label/
  save_epoch_step: 4
  eval_batch_step: [10000, 5000]
  train_batch_size_per_card: 8
  test_batch_size_per_card: 1
  image_shape: [3, 640, 640]
  reader_yml: ./configs/det/det_db_bankcard_v5_no_check_label_reader.yml
  pretrain_weights: ./pretrain_models/ch_ppocr_server_v1.1_det_train/
  save_res_path: ./output/det_r18_vd_db_bankcard_v5_no_check_label/predicts_db.txt
  checkpoints:
  save_inference_dir: ./inference/det_db_bankcard_v5_no_check_label

Architecture:
  function: ppocr.modeling.architectures.det_model,DetModel

Backbone:
  function: ppocr.modeling.backbones.det_resnet_vd,ResNet
  layers: 18

Head:
  function: ppocr.modeling.heads.det_db_head,DBHead
  model_name: large
  k: 50
  inner_channels: 256
  out_channels: 2

Loss:
  function: ppocr.modeling.losses.det_db_loss,DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  function: ppocr.optimizer,AdamDecay
  base_lr: 0.001
  beta1: 0.9
  beta2: 0.999
  decay:                                                                                                                         
    function: cosine_decay_warmup
    step_each_epoch: 32                                                                                                         
    total_epoch: 1200

PostProcess:
  function: ppocr.postprocess.db_postprocess,DBPostProcess
  thresh: 0.3
  box_thresh: 0.5
  max_candidates: 1000
  unclip_ratio: 1.6
  
#cat ./configs/det/det_db_bankcard_v5_no_check_label_reader.yml
TrainReader:
  reader_function: ppocr.data.det.dataset_traversal,TrainReader
  process_function: ppocr.data.det.db_process,DBProcessTrain
  num_workers: 1
  img_set_dir: ./train_data/bankcard/text_localization/bankcard_det_v5_no_check_label/ppocrlabel_bankcard/
  label_file_path: ./train_data/bankcard/text_localization/bankcard_det_v5_no_check_label/train_bankcard_label.txt

EvalReader:
  reader_function: ppocr.data.det.dataset_traversal,EvalTestReader
  process_function: ppocr.data.det.db_process,DBProcessTest
  img_set_dir: ./train_data/bankcard/text_localization/bankcard_det_v5_no_check_label/ppocrlabel_bankcard/
  label_file_path: ./train_data/bankcard/text_localization/bankcard_det_v5_no_check_label/test_bankcard_label.txt
  #test_image_shape: [736, 1280]
  test_image_shape: [640, 640]
  
TestReader:
  reader_function: ppocr.data.det.dataset_traversal,EvalTestReader
  process_function: ppocr.data.det.db_process,DBProcessTest
  infer_img:
  img_set_dir: ./train_data/bankcard/text_localization/bankcard_det_v5_no_check_label/ppocrlabel_bankcard/
  label_file_path: ./train_data/bankcard/text_localization/bankcard_det_v5_no_check_label/test_bankcard_label.txt
  do_eval: True

**Comment by yisampi** [comment]
@WenmuZhou 还有一个问题 就是训练程序还在 但是没有log输出了
![Uploading image.png…]()

**Comment by yisampi** [comment]
@WenmuZhou 这个问题还经常发生的？有解决办法吗？谢谢

**Comment by WenmuZhou** [comment]
你这个图片看不了，你也可以用dtgraph分支的代码试一下
